project1/ConstantBSProcess.h

```diff
@@ -0,0 +1,27 @@
+#include <cmath>
+#include <algorithm>
+#include <stochasticprocess.hpp>
```

Comment:
This should be
```suggestion
#include <ql/stochasticprocess.hpp>
```
so that it finds the one inside your copy of QuantLib.  If it works, you can remove the `stochasticprocess.*` files in this directory.